src/components/Input/Input.tsx

```diff
@@ -104,120 +110,119 @@ class Input extends AutoControlledComponent<Extendable<IInputProps>, any> {
      */
     renderInput: PropTypes.func,
 
-    /** Additional CSS styles to apply to the component instance.  */
+    /**
+     * A custom render function the wrapper slot.
+     *
+     * @param { React.ReactType } Component - The computed component for this slot.
+     * @param { object } props - The computed props for this slot.
+     * @param { ReactNode | ReactNodeArray } children - The computed children for this slot.
+     */
+    renderWrapper: PropTypes.func,
+
+    /** Additional CSS styles to apply to the component instance. */
     styles: PropTypes.oneOfType([PropTypes.object, PropTypes.func]),
 
+    /** The HTML input type. */
+    type: PropTypes.string,
+
     /** The value of the input. */
     value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
 
     /** Override for theme site variables to allow modifications of component styling via themes. */
     variables: PropTypes.oneOfType([PropTypes.object, PropTypes.func]),
+
+    /** Shorthand for the wrapper component. */
+    wrapper: customPropTypes.itemShorthand,
```

Comment:
note that now, with this Prop type being defined in a way it is, we have the following error in console output when component is rendered:

> Failed prop type: Prop `wrapper` in `Input` conflicts with props: `children`. They cannot be defined together, choose one or the other.

Reply:
fixing

Reply:
done